adi/__init__.py

```diff
@@ -103,6 +103,7 @@
 from adi.one_bit_adc_dac import one_bit_adc_dac
 from adi.QuadMxFE_multi import QuadMxFE_multi
 from adi.tdd import tdd
+from adi.cn0556 import cn0556
```

Comment:
Move this to the correct line or run pre-commit to sort the imports.

Reply:
Moved to correct line.